crates/forge-runner/src/test_case_summary.rs

```diff
@@ -27,6 +27,8 @@ pub enum TestCaseSummary {
         fuzzing_statistic: Option<FuzzingStatistics>,
         /// Number of block used if BlockId::Tag(Latest) was specified
         latest_block_number: Option<BlockNumber>,
+        /// Gas used by the test case
+        gas: f64,
```

Comment:
Not requesting a change, just curious: Why is f64 used for gas. Are there things that have fractional gas cost?

Reply:
Yup, like everything
https://book.starknet.io/ch03-01-02-fee-mechanism.html#computation